dts/bindings/led_strip/worldsemi,ws2812-i2s.yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 Jonathan Rico
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Worldsemi WS2812 LED strip, I2S binding
+
+  Driver bindings for controlling a WS2812 or compatible LED
+  strip with an I2S master.
+
+compatible: "worldsemi,ws2812-i2s"
+
+include: [base.yaml, ws2812.yaml]
+
+properties:
+
+  i2s-dev:
+    type: phandle
+    required: true
+    description: Pointer to the I2S instance.
+
+  out-active-low:
+    type: boolean
+    required: false
```

Comment:
Remove this. False is the default and true would not make sense for booleans.

Reply:
yeah boolean handling in dts is still a bit weird to me